src/cassandra/result.rs

```diff
@@ -186,13 +186,13 @@ impl<'a> Iterator for ResultIterator<'a> {
 
 impl<'a> ResultIterator<'a> {
     /// Gets the next row in the result set
-    pub fn get_row(&mut self) -> Row {
+    pub fn get_row(&mut self) -> Row<'a> {
         unsafe { Row::build(cass_iterator_get_row(self.0)) }
     }
 }
 
 impl<'a> IntoIterator for &'a CassResult {
-    type Item = Row;
+    type Item = Row<'a>;
```

Comment:
Cool, this is the key. Thanks. Interesting that the `<'a>` was already available on the `ResultIterator`, just not used - so someone was aware of the issue. Nice.